src/lib/cdk-accelerator/src/codebuild/cdk-deploy-project.ts

```diff
@@ -131,7 +131,7 @@ export class PrebuiltCdkDeployProject extends CdkDeployProjectBase {
     fs.writeFileSync(
       path.join(this.projectTmpDir, 'Dockerfile'),
       [
-        'FROM public.ecr.aws/bitnami/node:12',
+        'FROM node:12.21-slim as builder',
```

Comment:
We just switched away from docker registry due to throttling issues based on their policy changes. No way to reliably get around this problem.  We need to provide customers with reliable error free installs.

Reply:
While moving to different baserepo only my image size reduced from 1.12Gb to 737MB.  We can definitely clone the baserepo in ecr gallery

Reply:
Also this is one big thing which I am unable to understand of AWS, aws provided public registry, a common expectation is to allow people move their workloads to ecr gallery from dockerhub. Would be nice if it can act as a caching in the first place. For aws its just a little trick of modifying the dns resolution and serve like a vanilla maven cache.

We can mirror the node binary to bitnami one too which is available in ecr.

Reply:
I think I mentioned on one of your other tickets we are exploring options in this regard and considering changing approaches.  The original design principal was everything would be built on initial install such that nobody would have concerns about what was installed on a custom image and everything was 100% transparent.  Not prepared to include this change in this specific PR.

Reply:
Reverted